apps/web/src/components/auth/signup-form.tsx

```diff
@@ -83,6 +106,52 @@ export default function SignUpForm() {
                             </FormItem>
                         )}
                     />
+                    <FormField
+                        control={form.control}
+                        name="electricityMeterType"
+                        render={({ field }) => (
+                            <FormItem>
+                                <Select onValueChange={field.onChange} value={field.value}>
+                                    <FormControl>
+                                        <SelectTrigger>
+                                            <SelectValue placeholder="Wähle deinen Stromzähler" />
```

Comment:
Wir wollten ja einheitlich Siezen
```suggestion
                                            <SelectValue placeholder="Wählen Sie Ihren Stromzähler" />
```